models/base_menu.yml

```diff
@@ -12,7 +12,7 @@ spec:
           - namespace: Organization
             name: Manufacturer
             label: Manufacturer
-            kind: LocationManufacturer
```

Comment:
this has been fixed in 2 PRs already :)

Reply:
😂 I've reverted this so there's no conflicts.